src/components/resources/ResourcesSelect.jsx

```diff
@@ -0,0 +1,36 @@
+import { Grid, FormControl, InputLabel, Select } from "@mui/material";
+
+const ResourcesSelect = ({
+  children,
+  value,
+  name,
+  onChange,
+  label,
+  defaultValue,
+}) => {
+  return (
+    <Grid item xs={12} sm={12} md={5} lg={5}>
```

Comment:
Remove Grid component here to make input control reusable for other pages or components